ci/install-build-deps.sh

```diff
@@ -66,7 +66,8 @@ case "$ID-$VERSION_ID" in
       # an older version of a package must be used for a certain Python version.
       # If that information is not read, pip installs the latest version, which
       # then fails to run.
-      $SUDO_CMD pip3 install -U setuptools pip six
+      $SUDO_CMD pip3 install -U pip six
+      $SUDO_CMD pip3 install -U 'setuptools != 50.0.0'
```

Comment:
I wouldn't be surprised if they release another 50.x version without a full fix; how about `setuptools < 50` until we know it's fixed? And please add a comment as to why this is necessary.

Reply:
Makes sense. Shall do.